packages/extensions/venia-sample-language-packs/i18n/fr_FR.json

```
@@ -0,0 +1,366 @@
+{
+    "About Us": "À propos de nous",
+    "Account": "Compte",
+    "accountChip.chipText": "Bonjour, {name}",
+    "accountInformationPage.accountInformation": "Informations sur le compte",
+    "accountInformationPage.editAccount": "Modifier les informations du compte",
+    "accountInformationPage.errorTryAgain": "Un problème est survenu. Veuillez actualiser et réessayer.",
+    "accountInformationPage.titleAccount": "Information sur le compte - {name}",
+    "accountMenu.accountInfoLink": "Information sur le compte",
+    "accountMenu.addressBookLink": "Carnet d'adresses",
+    "accountMenu.communicationsLink": "Les communications",
+    "accountMenu.favoritesListsLink": "Listes de favoris",
+    "accountMenu.orderHistoryLink": "Historique des commandes",
+    "accountMenu.savedPaymentsLink": "Paiements enregistrés",
+    "accountMenu.signOutButtonText": "Déconnexion",
+    "accountMenu.storeCreditLink": "Cartes de crédit et cadeaux en magasin",
+    "accountTrigger.ariaLabel": "Toggle Menu Mon compte",
+    "accountTrigger.buttonFallback": "Se connecter",
+    "accountTrigger.signIn": "Se connecter",
+    "addressBook.addNewAddresstext": "Ajouter une nouvelle adresse",
+    "addressBook.applyButtonText": "Appliquer",
+    "addressBook.cancelButtonText": "Annuler",
+    "addressBook.headerText": "Modifier les informations de livraison",
+    "addressBookPage.addAddressText": "Ajouter une adresse",
+    "addressBookPage.addressBookText": "Carnet d'adresses",
+    "addressCard.defaultText": "Défaut",
+    "app.errorOffline": "Vous êtes hors ligne. Certaines fonctionnalités peuvent ne pas être disponibles.",
+    "app.errorUnexpected": "Pardon! Une erreur inattendue est apparue.",
+    "app.infoOnline": "Tu es en ligne.",
```

Given that you changed the `errorOffline` conjugation to be more polite, we probably want to make the same change throughout.

```suggestion
    "app.infoOnline": "Vous êtes en ligne.",
```